### PR TITLE
Added metadata for multipolygons,py file

### PR DESCRIPTION
**Pull request by Med16-11**
- [x] issues: fixes #11765
While going through the code base, I felt like it would be great if we add metadata for files other than plotting and models.
Just a thought from my side. Would like to know what you all think.

### CODE REVIEW

File: examples/basic/areas/multipolygons.py [hunk 1]
```diff
@@ -1,5 +1,11 @@
 from bokeh.plotting import figure, show
+"""
+It creates a multi-polygon plot using the Bokeh library. The plot has a width and height of 400 pixels each. 
+It consists of two sets of polygons, where each set is defined by its x and y coordinates. 
+The x and y coordinates are lists of lists, where each inner list contains the coordinates for one polygon. 
+The colors for the polygons are specified as a list of color strings. Finally, the plot is displayed using the show() function from Bokeh.
 
+"""
```
Comment (bryevdv) [inline examples/basic/areas/multipolygons.py]: A couple of issues to resolve:

* the docstring needs to come first in the file (move the imports below)
* for consistency with other examples, shorten docstring line length to 80-100 characters
* I would replace "It" with "The plot" in the text above

Lastly, the actual work is add a `bokeh-example-metadata` directive, which missing here. Here is an example you can refer to: 

https://github.com/bokeh/bokeh/blob/74b1f1a227dd1aa4735caa13199e79986b7e67f5/examples/basic/bars/colors.py#L4-L7

**Review by bryevdv — CHANGES_REQUESTED** [review]
Thanks for the PR @Med16-11 there are a few things to resolve, please let us know if we can provide additional guidance

File: examples/basic/areas/multipolygons.py (hunk 1 above)
Comment (Med16-11) [reply examples/basic/areas/multipolygons.py]: Thanks for the review Bryan! I will make the suggested changes.

Comment (Med16-11) [reply examples/basic/areas/multipolygons.py]: Hey ! @bryevdv , I realised that there is no requirement of metadata for this file. But we can have docstring for this file. Do let me know what do you think?

**Pull request closed without merging**